File: javascript/driveItem_createUploadSession.js
var data = "{ }";

var xhr = new XMLHttpRequest();
xhr.withCredentials = true;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === this.DONE) {
    console.log(this.responseText);
  }
});

xhr.open("POST", "https://graph.microsoft.com/v1.0/me/drive/root:/%7Bitem-path%7D:/createUploadSession");
xhr.setRequestHeader("content-type", "application/json");

xhr.send(data);

File: python/driveItem_createUploadSession.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("POST", "/v1.0/me/drive/root:/%7Bitem-path%7D:/createUploadSession", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
